### ISSUE TITLE
why did 45% get PASS but a 53.40% net from almost a week ago was rejected?

### ISSUE THREAD

**Issue by waterloo588** [gcp/leela-zero#1665]
why did 45% get PASS but a 53.40% net from almost a week ago was rejected?

https://archive.is/zWlC8

**Comment by MartinDevelopment** [comment]
We are now using a 256x20 network instead of a 192x15 network.

I hope someone can explain to me why this choice was made, since we are now using a network that can't even beat the 192x15 network with more resources and we are using that to create games at half the playouts.

**Comment by TFiFiE** [comment]
See #1113 and #1625.

**Comment by waterloo588** [comment]
I thought that guy had made a 40 by 256 and was going to release it for use after the competition was over? why going to a 20 block and not even a 30 block, makes no sense whatsoever since ELF v2 is likely a 40block and on 20b LZ won't ever catch up to the first version of ELF

**Comment by l1t1** [comment]
https://github.com/gcp/leela-zero/issues/1625#issuecomment-408070109 suggest to use elf weight as next best

@gcp It's not supported by the training code.

**Comment by jkiliani** [comment]
@gcp Is the learning rate reset to what we used for 192x15 in the beginning?

**Comment by alphaladder** [comment]
keep patience.... it has great potentiality.

**Comment by alphaladder** [comment]
@gcp just a reminder,please add the 274k self play game of the weight 157 to the training of new 158,because bjiyxo only use the games before 157 to obtain this v20-2,therefore the level of v20-2 should be the same as 0503.

**Comment by Splee99** [comment]
Back to the classroom for learning ladder again:
http://zero.sjeng.org/view/e49ac9169109174d80568c7b36bff13cce725df847c2bd94abd1d5631eb198b7?viewer=wgo

**Comment by gcp** [comment]
See the other topics regarding the 256x20 nets.

**Comment by gcp** [comment]
>just a reminder,please add the 274k self play games of the weight 157 to the training window

I don't do anything, I'm just letting the regular training procedure run as always.
